scalafix-core/shared/src/main/scala/scalafix/v0/Database.scala

```diff
@@ -0,0 +1,27 @@
+package scalafix.v0
+
+import scala.compat.Platform.EOL
+import scala.meta.internal.io.PathIO
+
+final case class Database(documents: Seq[Document]) {
```

Comment:
This is copy/pasted from Scalameta 3.x, right? If that's the case, it would be a good idea to update NOTICE.md.

Reply:
Good idea, done.